### PR TITLE
Add holochain state dump functionality

### PR DESCRIPTION
**Pull request by timotree3**
Adds a new route `host/tmp/state_dump` that will be used during scalability testing to ensure peer consistency.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -64,7 +64,9 @@ conductor:
 stop-conductor:
 	yarn run stop-conductor
 
-test-integration:	build DNAs stop-lair lair
+test-integration:	build DNAs
+	make stop-lair
```
Comment (zo-el) [inline Makefile:68]: was there a reason for this change?

File: src/index.ts [hunk 2]
```diff
@@ -551,6 +551,33 @@ class Envoy {
       return response_message;
     }, this.opts.NS);
 
+    // Temporary "state dump" logic for checking Holochain network peer consistency
+    this.ws_server.register("holo/tmp/state_dump", async ({ installed_app_id, cell_nick }) => {
```
Comment (zo-el) [inline src/index.ts]: Why `/tmp/` are we going to be removing it?

File: Makefile (hunk 1 above)
Comment (timotree3) [reply Makefile:68]: make jobs are run in parallel

File: src/index.ts (hunk 2 above)
Comment (timotree3) [reply src/index.ts]: I assume so. We don't want anybody who can spin up a chaperone to be able to see the state dump of anybody else's holochain cells

**Review by zo-el — APPROVED** [review]
Looks good, just left my concern about the being a tmp solution in the chaperone PR

**Pull request merged**